src/main/resources/hudson/security/GlobalMatrixAuthorizationStrategy/config.jelly

```diff
@@ -182,28 +184,26 @@ THE SOFTWARE.
       </tr>
     </table>
     <local:isEditable>
-      <div class="jenkins-!-margin-top-2">
-        <input type="button" class="matrix-auth-add-button" value="${%Add user…}" id="${id}UserButton"
+      <div class="jenkins-!-margin-top-2 jenkins-buttons-row jenkins-buttons-row--equal-width">
+        <button type="button" class="jenkins-button matrix-auth-add-button" id="${id}UserButton"
```

Comment:
causes a [regression in the acceptance-test-harness](https://github.com/jenkinsci/acceptance-test-harness/issues/1286).